nautobot/core/tests/integration/test_plugin_home.py

```diff
@@ -0,0 +1,123 @@
+from django.test.utils import override_settings
+from django.conf import settings
+from unittest import skipIf
+
+from nautobot.circuits.models import Circuit, Provider
+from nautobot.dcim.models import PowerFeed, PowerPanel, Site
+from nautobot.tenancy.models import Tenant
+from nautobot.utilities.testing.integration import SeleniumTestCase
+
+from dummy_plugin.models import DummyModel
+from dummy_plugin.homepage import get_dummy_data
+
+
+@skipIf(
+    "dummy_plugin" not in settings.PLUGINS,
+    "dummy_plugin not in settings.PLUGINS",
+)
+class PluginHomeTestCase(SeleniumTestCase):
+    """Integration test the plugin homepage."""
+
+    fixtures = ["user-data.json"]  # bob/bob
+    layout = {
+        "Organization": {
+            "Sites": {"model": Site, "permission": "dcim.view_site"},
+            "Dummy Models": {"model": DummyModel, "permission": "dummy_plugin.view_dummymodel"},
+            "Tenant": {"model": Tenant, "permission": "tenancy.view_tenant"},
+        },
+        "Dummy Plugin": {
+            "Dummy Models": {"model": DummyModel, "permission": "dummy_plugin.view_dummymodel"},
+        },
+        "Power": {
+            "Power Feeds": {"model": PowerFeed, "permission": "dcim.view_powerfeed"},
+            "Power Panel": {"model": PowerPanel, "permission": "dcim.view_powerpanel"},
+        },
+        "Circuits": {
+            "Providers": {"model": Provider, "permission": "circuits.view_provider"},
+            "Circuits": {"model": Circuit, "permission": "circuits.view_circuit"},
+        },
```

Comment:
Doesn't look like these tests are doing anything with the `model` or `permission` keys?

Reply:
Models are being used in `test_homepage_render_counters`. I'm going to write another test now to use the permissions.